modules/batch-tool/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.devonfw.java.dev</groupId>
+		<artifactId>devon4j-modules</artifactId>
+		<version>dev-SNAPSHOT</version>
+	</parent>
+
+	<groupId>com.devonfw.java.modules</groupId>
+	<artifactId>devon4j-batch-tool</artifactId>
+	<version>${devon4j.version}</version>
+	<packaging>jar</packaging>
+	<name>${project.artifactId}</name>
+	<description>Tools to manage spring batch jobs.</description>
+
+	<dependencies>
+		<!-- devon4j modules -->
+		<dependency>
+			<groupId>com.devonfw.java.modules</groupId>
+			<artifactId>devon4j-logging</artifactId>
+		</dependency>
+
+
+		<!-- Spring Batch -->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-batch</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<scope>runtime</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>info.picocli</groupId>
+			<artifactId>picocli-spring-boot-starter</artifactId>
+			<version>4.1.4</version>
+		</dependency>
+
+	</dependencies>
+
+	<build>
+	  <finalName>devonfw-batch-tool</finalName>
```

Comment:
should be named `devon4j-batch-tool` as it belongs to this `devon4j` stack and all modules and artifacts are named `devon4j-*`.

Reply:
changed it with latest commit